### PR TITLE
Feat/open api doc feature

### PR DESCRIPTION
**Pull request by Sc0ra**

### CODE REVIEW

File: packages/serverless-contracts/src/features/__tests__/openApiDocumentation.test.ts [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+      id: 'testContract2',
+      path: '/users',
+      method: 'POST',
+      integrationType: 'httpApi',
+      bodySchema,
+      outputSchemas: outputSchemas2,
+    });
+
+    it('should generate open api documentation', () => {
+      console.log(
```
Comment (adriencaccia) [inline packages/serverless-contracts/src/features/__tests__/openApiDocumentation.test.ts:89]: You can remove this

**Review by adriencaccia — CHANGES_REQUESTED** [review]
Huge work @Sc0ra 🎉 
A really small change and we're good

**Review by fargito — APPROVED** [review]
Huge work, thanks @Sc0ra!

**Review by adriencaccia — APPROVED** [review]

File: user-docs/documentation/docs/how-to-guides/1-use-serverless-contracts/1-api-gateway.md [hunk 2]
```diff
@@ -424,6 +424,126 @@ const [mockEvent2] = getMockHandlerInput(myContract);
...
+import { getOpenApiDocument } from '@swarmion/serverless-contracts';
+
+const openApiDocumentation = getOpenApiDocumentation({
+  title: 'Test API',
+  description: 'API description',
+  contracts: [getUserContract, postUserContract, deleteUserContract],
+});
+```
+
+The resulting value will be something like this OpenAPI documentation (depending on your contracts)
```
Comment (adriencaccia) [inline user-docs/documentation/docs/how-to-guides/1-use-serverless-contracts/1-api-gateway.md:441]: Can you state that the result will be a JSON object?

I fear that the fact that you showcase `yml` might be misleading for the readers.

**Review by adriencaccia — CHANGES_REQUESTED** [review]

**Review by adriencaccia — APPROVED** [review]

**Pull request merged**